bip-musig2/gen_vectors_helper.py

```diff
@@ -127,7 +127,8 @@ def sig_agg_vectors():
         session_ctx = SessionContext(aggnonce, vec_pubkeys, vec_tweaks, is_xonly, msg)
 
         for j in range(len(key_indices)):
-            psigs[psig_indices[j]] = sign(secnonces[nonce_indices[j]], sk[key_indices[j]], session_ctx)
+            secnonce_tmp = bytearray(secnonces[nonce_indices[j]])
+            psigs[psig_indices[j]] = sign(secnonce_tmp, sk[key_indices[j]], session_ctx)
```

Comment:
nit: add the "don't copy" warning also here?

Reply:
done